### PR TITLE
Update README.md for unit tests

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue # .

### Summary of Changes
Added the description for a file that needs to be changed.

### Testing Instructions
Code review

### CODE REVIEW

File: tests/Unit/README.md [hunk 1]
```diff
@@ -8,4 +8,18 @@ When you are checking out the current development branch of 4.0 and run `compose
 
 1. Checkout the current Joomla 4.0 development branch from Github. (https://github.com/joomla/joomla-cms.git Branch `4.0-dev`)
 2. Run `composer install` in the root of your checkout.
-3. Run `libraries/vendor/bin/phpunit`.
+3. Adjust the file `/joomla-cms/phpunit-pgsql.xml.dist` und/oder `joomla-cms/phpunit-pgsql.xml.dist` based on your environment.
```
Comment (brianteeman) [inline tests/Unit/README.md]: In english please ;) I am guessing it should be 
```suggestion
3. Edit the file `/joomla-cms/phpunit-pgsql.xml.dist` and/or `joomla-cms/phpunit-pgsql.xml.dist` depending on your environment.
```

Comment (astridx) [reply tests/Unit/README.md]: Thanks

File: tests/Unit/README.md [hunk 2]
```diff
@@ -8,4 +8,18 @@ When you are checking out the current development branch of 4.0 and run `compose
 
 1. Checkout the current Joomla 4.0 development branch from Github. (https://github.com/joomla/joomla-cms.git Branch `4.0-dev`)
 2. Run `composer install` in the root of your checkout.
-3. Run `libraries/vendor/bin/phpunit`.
+3. Edit the file `/joomla-cms/phpunit-pgsql.xml.dist` and/or `joomla-cms/phpunit-pgsql.xml.dist` depending on your environment.
```
Comment (richard67) [inline tests/Unit/README.md]:
```suggestion
3. Edit the file `joomla-cms/phpunit.xml.dist` and/or `joomla-cms/phpunit-pgsql.xml.dist` depending on your environment.
```

**Comment by astridx** [comment]
Thank you Richard. I just saw that something else was wrong.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 55244cf7c6db2ef9011b63fc8f984c40c18fc8bd<br /><br />1. Code review: Ok.
2. Real test: Without a phpunit.xml, database tests fail because no database found. After proceeding as adviced in this PR, unit tests worked.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32758">issues.joomla.org/tracker/joomla-cms/32758</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 55244cf7c6db2ef9011b63fc8f984c40c18fc8bd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32758">issues.joomla.org/tracker/joomla-cms/32758</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32758">issues.joomla.org/tracker/joomla-cms/32758</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by richard67** [comment]
@astridx @wilsonge Sorry, seems I was sleeping when reviewing this PR. I've found some issues, see #32935 . Please review.

**Pull request merged**